cgi-bin/LJ/Faq.pm

```diff
@@ -98,6 +101,16 @@ sub load {
         return undef unless $f;
         $faq = $class->new(%$f, lang => $lang);
         $faq->lang_update_in_place;
+    } else {
+        my $f = $dbr->selectrow_hashref
+                ("SELECT faqid, question, summary, answer, faqcat, lastmoduserid, ".
+                 "DATE_FORMAT(lastmodtime, '%M %D, %Y') AS lastmodtime, ".
+                 "UNIX_TIMESTAMP(lastmodtime) AS unixmodtime, sortorder ".
+                 "FROM faqhist WHERE faqid=? AND rev=?",
+                 undef, $faqid, $rev);
+        die $dbr->errstr if $dbr->err;
+        return undef unless $f;
+        $faq = $class->new(%$f, lang => $lang);
```

Comment:
I'm assuming this part has to be revised to deal with calling lang_update_in_place since it's supposed to be loading a faq in something other than the site's default language, given its position at the end of the if statement.